configs/sgmoe_validator_sample_config.txt

```diff
@@ -0,0 +1,52 @@
+dataset.batch_size: 10
+dataset.block_size: 20
+dataset.data_dir: ~/.bittensor/data/
+dataset.dataset_name: default
+dataset.max_corpus_size: 10000.0
+dataset.max_datasets: 3
+dataset.no_tokenizer: false
+dataset.num_workers: 0
+dataset.save_dataset: false
+
+dendrite.max_active_receptors: 500
+dendrite.max_worker_threads: 150
+dendrite.requires_grad: true
+dendrite.timeout: 12
+
+logging.debug: false
+logging.logging_dir: ~/.bittensor/miners
+logging.record_log: false
+logging.trace: false
+
+neuron.blocks_per_epoch: 100
+neuron.clip_gradients: 1.0
+neuron.device: cpu
+neuron.learning_rate: 0.0001
+neuron.learning_rate_chain: 1
+neuron.metagraph_sync: 5000
+neuron.momentum: 0.8
+neuron.n_topk_peer_weights: 100
+neuron.name: sgmoe_validator
+neuron.no_restart: false
+neuron.topk: 20
+
+nucleus.dropout: 0.2
+nucleus.importance: 0.01
+nucleus.nhead: 2
+nucleus.nhid: 200
+nucleus.nlayers: 2
+
+subtensor.chain_endpoint: null
+subtensor.network: local
+
+wallet.hotkey: ndubuisi
+wallet.name: ndubuisi
```

Comment:
Make sure your don't have your information in the configs

Reply:
oof how did i miss that lol